### ISSUE TITLE
Keep track of N forecasts submitted, N skipped

### ISSUE THREAD

**Issue by rickyegeland** [ktindiana/sphinxval#42]
It should be possible from the SPHINX reports to determine how many forecasts of a given short name were submitted to SPHINX, and how many were skipped e.g. in the All Clear quantity.  This can help reveal unexpected omissions or errors in the processing.

The `N_submitted` and `N_skipped` values should  appear in both the `*_metrics.csv` files and in the md/html reports.

**Comment by rickyegeland** [comment]
@ktindiana made some progress on this recently, right?  Which commit?

**Comment by ktindiana** [comment]
@rickyegeland No, the code does not report this information yet. As a human, I looked at files and understood a way to check this, but no coding has been done.
